src/sdam/server.ts

```diff
@@ -397,9 +462,10 @@ Object.defineProperty(Server.prototype, 'clusterTime', {
 
 function supportsRetryableWrites(server: Server) {
   return (
-    server.description.maxWireVersion >= 6 &&
-    server.description.logicalSessionTimeoutMinutes &&
-    server.description.type !== ServerType.Standalone
+    server.loadBalanced ||
```

Comment:
Moved to utils, but there is still a weird typescript error that forced me to show the function's return type as `boolean`, `number` or `undefined`, which I still need to sort out.

Reply:
I think it's because we are referencing the actual properties and not doing `!!` on the checks, I'm leaving a comment to that end on the utils file and resolving this conversation